04/app.js

```diff
@@ -2,7 +2,64 @@ console.log('DOM');
 
 // struktura do wykorzystania w pętli
 const menuItems = [
-    {text: 'start', url: '/'},
-    {text: 'galeria', url: '/gallery'},
-    {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+    { text: 'start', url: '/' },
+    { text: 'galeria', url: '/gallery' },
+    { text: 'kontakt', url: '/contact' },
+];
+
+const nav = document.querySelector('nav');
+
+// if (nav) {
+//     const navList = document.createElement('ul');
+//     nav.appendChild(navList);
+//     const navItemFirst = document.createElement('li');
+//     const navItemSecond = document.createElement('li');
+//     const navItemThird = document.createElement('li');
+//     const navLinkFirst = document.createElement('a');
+//     const navLinkSecond = document.createElement('a');
+//     const navLinkThird = document.createElement('a');
+
+//     navList.append(navItemFirst, navItemSecond, navItemThird);
+//     navLinkFirst.setAttribute('href', '/')
+//     navLinkFirst.innerText = 'start';
+//     navLinkSecond.setAttribute('href', '/gallery')
+//     navLinkSecond.innerText = 'galeria';
+//     navLinkThird.setAttribute('href', '/contact')
+//     navLinkThird.innerText = 'kontakt';
+
+//     navItemFirst.appendChild(navLinkFirst);
+//     navItemSecond.appendChild(navLinkSecond);
+//     navItemThird.appendChild(navLinkThird);
+// }
+
+
+if (nav) {
+    const navList = document.createElement('ul');
+    nav.appendChild(navList);
+
+    // Create <li> array
+    const navItemList = [];
+    for (let i = 0; i < 3; i++) {
```

Comment:
Wpisywanie `3` zamiast `menuItems.length` jest złym pomysłem bo dodanie kolejnego elementu w `menuItems` powoduje, że musimy zmieniać w innym miejscu kod (pamiętanie o tym w dużym projekcie jest bardzo trudne).

Reply:
Faktycznie, na samym początku zrobiłem pętlę, która miała tworzyć "na sztywno" 3 elementy i zapomniałem to później podmienić – dzięki!